apps/mobile/src/state/storage-persistors.ts

```diff
@@ -137,18 +137,23 @@ export function useMnemonic({
   shouldLoadOnInit?: boolean;
 }) {
   const [mnemonic, setMnemonic] = useState<string | null>(null);
+  const [passphrase, setPassphrase] = useState<string | null>(null);
   async function getMnemonic() {
     const response = await mnemonicStore(fingerprint).getMnemonic();
     if (response.mnemonic) {
       setMnemonic(response.mnemonic);
     } else {
       throw new Error('No mnemonic found for this fingerprint');
     }
+
+    if (response.passphrase) {
+      setPassphrase(response.passphrase);
+    }
   }
   useEffect(() => {
     if (shouldLoadOnInit) {
```

Comment:
Does `useMnemonic` mean we always keep it in memory?

Reply:
I just made [this issue ](https://github.com/leather-io/issues/issues/315) for tracking this so we can merge this PR